foreman/Dockerfile

```diff
@@ -13,6 +13,12 @@ COPY data_models/dist/data-refinery-models-* data_models/
 RUN pip install \
```

Comment:
The latter, although if you think it'd be more clear and/or explicit if I used pip3/python3 in this file I can do that.